packages/fhir-converter/src/templates/cda/Header.hbs

```diff
@@ -79,14 +79,16 @@
             {{/with}}           
         {{/if}}
     {{/if}}
-            
-    {{#if document.author.assignedAuthor}}
-        {{#with (evaluate 'Utils/GeneratePractitionerId.hbs' obj=document.author.assignedAuthor) as |practitionerId|}}
-            {{>Resources/Practitioner.hbs practitioner=document.author.assignedAuthor ID=practitionerId.Id}},
+
+    {{#each (toArray document.author) as |author|}}
+      {{#if author}}
+          {{#with (evaluate 'Utils/GenerateId.hbs' obj=author.assignedAuthor) as |practitionerId|}}
+            {{>Resources/Practitioner.hbs practitioner=author.assignedAuthor ID=practitionerId.Id}},
             {{>References/Composition/author.hbs ID=(generateUUID (toJsonString document)) REF=(concat 'Practitioner/' practitionerId.Id)}},
-        {{/with}}
-    {{/if}}
-        
+          {{/with}}
```

Comment:
this is probably why we have more practitioners - was this intentional to move to looping through all authors?

Reply:
A Composition cannot be created without an author (required by schema). When i logged the document.author for xmls that had issue its because it was an array. So i looped through it to get all the assignedAuthors.